React/CoreModules/RCTAppearance.mm

```diff
@@ -165,7 +160,7 @@ - (void)appearanceChanged:(NSNotification *)notification
 
 - (void)startObserving
 {
-  if (@available(macOS 10.14, iOS 13.0, *)) { // TODO(macOS GH#774)
+  if (@available(iOS 13.0, *)) {
     [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
I need to verify that this still runs on macOS if I remove the macOS but of the availability check

Reply:
Still works!